ansible/roles/check_networking/tasks/main.yml

```diff
@@ -53,6 +53,13 @@
 # - debug: msg="ubuntu output is {{ ubuntu_remote_user.stdout }} and err is {{ ubuntu_remote_user.stderr }}"
 #   when: default_remote_user|failed and centos_remote_user|failed
 
+- name: fail when ssh errors occur related to Host key verification failed.
+  fail:
+    msg: "Configuration Error: Host key verification failed for IP: {{vm_ip}} . Does your ssh/config have 'StrictHostKeyChecking no' for the IP/IP Range/*"
+  when: (default_remote_user.stderr is defined and default_remote_user.stderr == "Host key verification failed.") and
+        (centos_remote_user.stderr is defined and centos_remote_user.stderr == "Host key verification failed.") and
+        (ubuntu_remote_user.stderr is defined and ubuntu_remote_user.stderr == "Host key verification failed.")
```

Comment:
🔎 Yes we should! It was required to move to `in` rather than `==` to catch things like `Host Key mismatch`! 👍